packages/code-du-travail-frontend/src/conventions/texte/SummaryItem.js

```diff
@@ -1,22 +1,24 @@
 import React from "react";
 import PropTypes from "prop-types";
-import styled, { css } from "styled-components";
+import styled from "styled-components";
 import { theme } from "@cdt/ui/";
+import { TocListItem, TocList, TocLink } from "./toc";
 
-const SummaryItem = ({ data, children, type, level, visible }) => {
```

Comment:
the visible property was actually not used, it was a leftover from a previous version that shouldn't have made it to master